addons/web/static/src/views/fields/copy_clipboard/copy_clipboard_field.js

```diff
@@ -13,16 +13,25 @@ const { Component } = owl;
 
 class CopyClipboardField extends Component {
     setup() {
-        this.copyText = this.env._t("Copy");
+        this.copyText = this.props.label || this.env._t("Copy");
         this.successText = this.env._t("Copied");
     }
     get copyButtonClassName() {
         return `o_btn_${this.props.type}_copy`;
     }
+    get getProps() {
+        return _.omit(this.props, "label");
```

Comment:
we don't use the lib _ anymore.
```suggestion
        const props = {...this.props}
        delete props.label
        return props
```